app/models/event.rb

```diff
@@ -37,30 +39,30 @@ def opened
     end
 
     def most_recent
-      order("created_at DESC").limit(3)
+      order('created_at DESC').limit(3)
     end
 
     def occurs_first
-      order("occurs_at ASC")
+      order('occurs_at ASC')
     end
 
     def active
-      where('occurs_at >= ?', Date.today)
+      where('occurs_at >= ?', Time.zone.today)
```

Comment:
Using Date.today will bring you the machine's datetime, which might differ from your actual datetime (with time zones)

Usually I switch from Date.today to Time.zone.today or Date.current.
source: https://robots.thoughtbot.com/its-about-time-zones

Though in this case Date.current might be better, due to Time's [limitations](http://stevenyue.com/2013/03/23/date-time-datetime-in-ruby-and-rails/):